pkg/types/openstack/platform.go

```diff
@@ -22,7 +22,10 @@ type Platform struct {
 	ExternalNetwork string `json:"externalNetwork,omitempty"`
 
 	// FlavorName is the name of the compute flavor to use for instances in this cluster.
-	FlavorName string `json:"computeFlavor"`
+	// Deprecated: this value is not used by the installer, use type in the defaultMachinePlatform section
```

Comment:
We need to continue to honor any values set in the field. That is the contract of deprecation. The installer cannot just ignore the field.

Reply:
we continue to honor any values set in this field. to do so I implemented the conversion and related tests: https://github.com/openshift/installer/pull/4321/files#diff-2bfaee7ebe030f992df86bdd39b047cfb12132e99ff453d2ddf4dc97c5d8530fR109-R122

Reply:
OK. Good. Can we change the comment to not state that the field is ignored?

Reply:
sure! I removed the part where it says that this value is not used by the installer